Common/Orders/OrderEvent.cs

```diff
@@ -33,11 +33,21 @@ public class OrderEvent
         public Symbol Symbol;
 
         /// <summary>
+        /// The date and time of this event (UTC).
+        /// </summary>
+        public DateTime Time;
+
+        /// <summary>
```

Comment:
Oops. You're absolutely right: orderEvent.UtcTime is the time the event is generated (that can be a fill or not ?) and order.Time is the order placement time. I'll fix it immediately.
I suspected order.Time could be used in algorithm, so I didn't touch it.